build_debian.sh

```diff
@@ -100,6 +98,9 @@ echo '[INFO] Mount all'
 ## Output all the mounted device for troubleshooting
 sudo LANG=C chroot $FILESYSTEM_ROOT mount
 
+## Install the trusted gpg public keys
+[ -d $TRUSTED_GPG_DIR ] && [ ! -z "$(ls $TRUSTED_GPG_DIR)" ] && sudo cp $TRUSTED_GPG_DIR/* ${FILESYSTEM_ROOT}/etc/apt/trusted.gpg.d/
```

Comment:
Is it better to install gpg file by `curl | sudo apt-key add -`.
There is an example in sonic-slave-buster/Dockerfile.j2

Reply:
See another comment in scrpts/prepare_docker_buildinfo.sh

---
In reply to: [539809352](https://github.com/Azure/sonic-buildimage/pull/5718#discussion_r539809352) [](ancestors = 539809352)